src/main/java/uc/doc/downloaddoc/DownloadDocOutputBoundary.java

```diff
@@ -13,6 +13,6 @@ public interface DownloadDocOutputBoundary {
     /** Prepares a failure view
      * @param errorMessage The errorMessage that occurs when a failure view happens
      */
-    DownloadDocRequestModel prepareFailureView(String errorMessage);
+    DownloadDocResponseModel prepareFailureView(String errorMessage);
```

Comment:
Good job catching this typo! I think the presenter needs updated for this too which is what's causing compilation issues